src/tests/unit/utils/mock-console.ts

```diff
@@ -0,0 +1,15 @@
+export default class MockConsole {
+  _buffer: string[];
+  log: (message: string) => void;
+
+  constructor() {
+    this._buffer = [];
+    this.log = message => {
```

Comment:
Can't we just define the method body in the class?

Reply:
Let's just leave this in for now until we convert to using `console-test-helpers`, as discussed in person.